source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -105,11 +135,13 @@ export default class PinCode extends Component<Props> {
               themeId={IDENTIFIERS.INPUT}
               skin={InputSkin}
               onChange={(number) => this.onChange(number, key)}
+              onKeyPress={(event) => this.onKeyPress(event, key)}
```

Comment:
I can't use backspace unless there's a value in the field. Please check what needs to be done to fix this (it might be on React-polymorph

Reply:
@aleksandardjordjeviciohk what about this one?